streams/src/test/java/org/apache/kafka/streams/state/internals/CachingWindowStoreTest.java

```diff
@@ -67,7 +67,7 @@
     @Before
     public void setUp() {
         keySchema = new WindowKeySchema();
-        final int retention = 30000;
+        final int retention = 60000;
```

Comment:
Why this?

Reply:
The tests incorrectly depended on windows being present after their retention time passed.